src/Core/src/Platform/iOS/MauiTextView.cs

```diff
@@ -161,5 +173,21 @@ void ShouldCenterVertically()
 				_ => new CGPoint(0, 0),
 			};
 		}
+
+		void UpdatePlaceholderFontSize(UIFont? value)
+		{
+			if (value != null)
```

Comment:
is not is preferred over != because operator can be overridden leading to possible bugs.
```suggestion
			if (value is not null)
```